<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modern Chat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="messageLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QLineEdit" name="messageInput">
        <property name="styleSheet">
         <string>
          QLineEdit {
              background-color: #F2F2F2;
              border: 2px solid #CCCCCC;
              border-radius: 15px;
              padding: 12px;
              font-size: 16px;
          }
          QLineEdit:focus {
              border: 2px solid #0057D9;
          }
         </string>
        </property>
        <property name="placeholderText">
         <string>Type your message here...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="styleSheet">
         <string>
          QPushButton {
              background-color: #0057D9;
              color: white;
              border-radius: 15px;
              padding: 12px 24px;
              font-size: 16px;
              font-weight: bold;
              margin-left: 10px;
          }
          QPushButton:hover {
              background-color: #0042A0;
          }
          QPushButton:pressed {
              background-color: #002C6C;
          }
         </string>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
